frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SharingPaneButton/SharingPaneButton.module.css

```diff
@@ -11,3 +11,11 @@
     --external-link-icon-color: var(--mb-base-color-blue-40);
   }
 }
+
+.Disabled.Container:not([data-css-specificity-hack="🤷"]) {
+  --mb-color-text-primary: var(--mb-color-text-tertiary);
```

Comment:
I don't know how comfortable I feel about this one. I think it might make more sense to just change the color or override it in some way. Your call, but I feel like this paradigm makes more sense when we're working with hover states and one-off semantic colors rather than `text-primary`

Reply:
It's hard to override the color that I want to override since it's rendered as `children`.

https://github.com/metabase/metabase/blob/004d246310f7ee1f924d2b0808400dda12d2f8c5/frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SharingPaneButton/SharingPaneButton.tsx#L51

If you have any better solution in mind feel free to suggest. I think this is a good compromise. I try to pass the disabled class directly to each text we want to dull the color, but the markup is really messy. I think doing this in the CSS like this is way more manageable.

I'll leave this as is for this reason.